docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -496,6 +496,14 @@ Check [releases.deckhouse.io](https://releases.deckhouse.io) for the current sta
       - To authenticate in a third-party registry, you need to use the `--source-login` and `--source-password` parameters;
    - `--images-bundle-chunk-size=N` — to specify the maximum file size (in GB) to split the image archive into. As a result of the operation, instead of a single file archive, a set of `.chunk` files will be created (e.g., `d8.tar.NNNN.chunk`). To upload images from such a set of files, specify the file name without the `.NNNN.chunk` suffix in the `d8 mirror push` command (e.g., `d8.tar` for files like `d8.tar.NNNN.chunk`).
 
+   Additional configuration options for the `d8 mirror` family of commands are available as environment variables:
+    - `HTTP_PROXY`/`HTTPS_PROXY` — proxy URLs for HTTP(S) requests unless overridden by `$NO_PROXY`;
```

Comment:
> proxy URLs
Is it a proxy URL or a list of proxy URLs?

Reply:
@mvasl

Reply:
Single url

Reply:
Offered the suggestion.